### PR TITLE
Merge master into feature-deviceconnect

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

**Comment by tranchitella** [comment]
Manually launched pipeline:
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/217872219

**Review by alfrunes — APPROVED** [review]

File: tests/tests/orchestrator.py [hunk 1]
```diff
@@ -25,11 +25,12 @@ def provision_device_handler(device_id=None, status=200):
     log = logging.getLogger("orchestartor.provision_device")
 
     def _provision_device(request):
```
Comment (alfrunes) [inline tests/tests/orchestrator.py:27]: I believe this is wrong, I updated the `provision_device` workflow parameter to contain a "device_id" property, not the entire Device

Comment (alfrunes) [reply tests/tests/orchestrator.py:27]: IOW; the diff should be the other way around.

Comment (tranchitella) [reply tests/tests/orchestrator.py]: RIght, fixed here: https://github.com/mendersoftware/deviceauth/pull/422/commits/a25729b4a119c48a79f583c9b3cb95607286dfdc

**Pull request merged**